src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs

```diff
@@ -98,11 +98,7 @@ void ICommandHandlerService.Execute<T>(IContentType contentType, T args, Action
             }
             else
             {
-                // If there aren't any command handlers, just invoke the last handler (if there is one).
-                if (lastHandler != null)
-                {
-                    lastHandler();
-                }
+                lastHandler?.Invoke();
```

Comment:
Looks like Cyrus's code fix doesn't handle trivia. Hey @cyrusnajmabadi, try using Matt's removenode API that attempts to leave trivia behind.